### ISSUE TITLE
Uncaught NullReferenceException thrown when loading a page with an AntDesign.Image component

### ISSUE THREAD

**Issue by tmeadon** [ant-design-blazor/ant-design-blazor#1141]
### Describe the bug

When loading a page containing an `<AntDesign.Image>` component (using the same code as in [the docs](https://antblazor.com/en-US/components/image)) the browser throws a `NullReferenceException`.

Code for component that raises the exception:

`<AntDesign.Image Width="200px" Src="https://zos.alipayobjects.com/rmsportal/jkjgkEfvpUPVyRjUImniVslZfWPnJuuZ.png"/>`

### Steps to reproduce (please include code)
Repro project: https://github.com/tommagumma/ant-blazor-image-bug

Issue can be reproduced by launching the project and loading `/image`.

### Exceptions (if any)
```
blazor.webassembly.js:1 crit: Microsoft.AspNetCore.Components.WebAssembly.Rendering.WebAssemblyRenderer[100]
      Unhandled exception rendering component: Object reference not set to an instance of an object.
System.NullReferenceException: Object reference not set to an instance of an object.
   at AntDesign.Image.BuildRenderTree(RenderTreeBuilder __builder)
   at Microsoft.AspNetCore.Components.ComponentBase.<.ctor>b__6_0(RenderTreeBuilder builder)
   at Microsoft.AspNetCore.Components.Rendering.ComponentState.RenderIntoBatch(RenderBatchBuilder batchBuilder, RenderFragment renderFragment)
   at Microsoft.AspNetCore.Components.RenderTree.Renderer.RenderInExistingBatch(RenderQueueEntry renderQueueEntry)
   at Microsoft.AspNetCore.Components.RenderTree.Renderer.ProcessRenderQueue()
```

### Further technical details
- AntDesign Nuget Package version: `0.6.0`

- Output of `dotnet --info`:

```
.NET SDK (reflecting any global.json):
 Version:   5.0.103
 Commit:    72dec52dbd

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.19042
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   C:\Program Files\dotnet\sdk\5.0.103\

Host (useful for support):
  Version: 5.0.3
  Commit:  c636bbdc8a

.NET SDKs installed:
  3.1.201 [C:\Program Files\dotnet\sdk]
  3.1.406 [C:\Program Files\dotnet\sdk]
  5.0.101 [C:\Program Files\dotnet\sdk]
  5.0.103 [C:\Program Files\dotnet\sdk]

.NET runtimes installed:
  Microsoft.AspNetCore.All 2.1.25 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.All]
  Microsoft.AspNetCore.App 2.1.25 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 3.1.3 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 3.1.12 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 5.0.3 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 2.1.25 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 3.1.3 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 3.1.12 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 5.0.3 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.WindowsDesktop.App 3.1.3 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 3.1.12 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 5.0.3 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]

To install additional .NET runtimes or SDKs:
  https://aka.ms/dotnet-download

```

- The IDE (VS / VS Code/ VS4Mac) you're running on, and it's version: VSCode version `1.53.2`

**Comment by ElderJames** [comment]
Thank you for contacting us @tommagumma 

You just missing the `<AntContainer />` for popup components. Please add  `<AntContainer />` in the `App.razor` file.

```
<Router AppAssembly="@typeof(Program).Assembly" PreferExactMatches="@true">
    <Found Context="routeData">
        <RouteView RouteData="@routeData" DefaultLayout="@typeof(MainLayout)" />
    </Found>
    <NotFound>
        <LayoutView Layout="@typeof(MainLayout)">
            <p>Sorry, there's nothing at this address.</p>
        </LayoutView>
    </NotFound>
</Router>

<AntDesign.AntContainer />
```

**Comment by ElderJames** [comment]
@Hona, could you please help to update your PR #1131 to include this in the docs?

**Comment by tommagumma** [comment]
@ElderJames - thank you for your quick response, it's much appreciated.  I've [added](https://github.com/tommagumma/ant-blazor-image-bug/commit/5fd0cfc191cdbe65dd3570aab2cdcc46ad07bb66) the missing component and am still receiving the `NullReferenceException`.

**Comment by tommagumma** [comment]
@ElderJames fyi - disabling the preview functionality (adding `Preview="false"`) prevents the exception from being raised.

**Comment by ElderJames** [comment]
Sorry, I can't reproduce the problem. Please try the latest nightly package.

**Comment by tommagumma** [comment]
@ElderJames the latest nightly has the same behaviour.  You weren't able to reproduce the problem using my repro project?

**Comment by ElderJames** [comment]
Yes, it works well here.

**Comment by anddrzejb** [comment]
I managed to recreate the problem. I tried your repro on chrome, edge and firefox - all was ok. Then I run on vivaldi - presto, the exception showed. What browser and version were you testing this on?

**Comment by tommagumma** [comment]
@anddrzejb - thanks for looking into this.  I've been testing this from Edge version 88.0.705.74.

**Comment by anddrzejb** [comment]
@tommagumma that is exactly the same version I was testing this on. And no error. No matter, there is clearly a bug there, that once @ElderJames decides the course of action, we are going to fix it.

**Comment by ElderJames** [comment]
I think, in the absence of a corresponding translation, it should be rolled back to en-US.

**Comment by anddrzejb** [comment]
The thing is, the translation may exist, but the part of json containing the image may not (as it is now). Unless you want to check every property if it exists in json and fall back to en-Us only for that missing part. If we are talking about en-US fallback, then the easiest and most full-proof solution is to hardcode it in `Locale.cs`, as I proposed in point 2.

**Comment by ElderJames** [comment]
I would prefer to handle it in `LocalProvider` and perhaps fix it along with the `IStringLocalizer` feature #1052 .

**Comment by anddrzejb** [comment]
So you mean in `GetLocale(string cultureName)` check each property of `Locale` and if it is null, fill it up from `en-Us.json` file, correct? The #1052 will probably require some discussion there.

**Comment by anddrzejb** [comment]
So either reflection (perf impact) or each property check hardcoded (maintenance impact). What's your poison?

**Comment by ElderJames** [comment]
fixed in #1710
